### DISCUSSION TITLE
intertemporal and Window Solve

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#97] [Q&A: Execution]
Morning ReEDS team!
I have encountered errors on my laptop using int and win solve.

In win solve, it creates "testmode_ref_win_0. list" and shows the error below.
**** The following LP error was detected in the model ReEDSmodel.
**** 66 equation eq_pvb_storage_in_max... The symbol "curt_old_pvb_lastyear" has no value specified.

In int solve, it stops at "testmode_ref_int_0.lst" and displays an error below:
5947376  cap_firm_iter(i,v,r,szn,t,"0")$storage(i) = sum{sdbin, CAP_SDBIN.l(i,v,r,szn,sdbin,t) * cc_storage(i,sdbin) } ;
****                                                                                                               $148

What can I do to fix these errors and get the model running? Thank you very much!

**Comment by wesleyjcole** [comment]
After releasing the 2021 model version we found some errors with the window and intertemporal solves.  If you need to use either of these methods for your work, please reach out to me at wesley.cole@nrel.gov.
